app/models/reviewable_score.rb

```diff
@@ -23,14 +23,15 @@ def self.type_title(type)
   # When extending post action flags, we need to call this method in order to
   # get the latests flags.
   def self.reload_types
-    @types = nil
+    @api_types = nil
     types
   end
 
   def self.add_new_types(type_names)
+    @api_types ||= {}
```

Comment:
At first I thought we had just pushed the problem to over here, but seems this is only used by a single, deprecated method, so should be fine? 🙂

Reply:
Yes, you are right. This method was deprecated in May 2024 and plugins maintained by us already migrated like for example https://github.com/discourse/discourse-antivirus/pull/57